src/pages/Search/components/SearchBar/RollingBanner.tsx

```diff
@@ -9,11 +9,31 @@ export default function RollingBanner() {
       ? <div />
       : (
         <div className={styles['search-rolling-banner']}>
-          <strong>TRENDING</strong>
           <ul className={styles['search-rolling-banner__tag-list']}>
-            <li className={styles['search-rolling-banner__tag']}>{trendings && trendings.map((tag) => `#${tag}`).join(' ')}</li>
-            <li className={styles['search-rolling-banner__tag']}>{trendings && trendings.map((tag) => `#${tag}`).join(' ')}</li>
-            <li className={styles['search-rolling-banner__tag']}>{trendings && trendings.map((tag) => `#${tag}`).join(' ')}</li>
+            <li className={styles['search-rolling-banner__tag']}>
+              {trendings && trendings.map((tag) => (
+                <span className={styles['search-rolling-banner__tag-style']}>
+                  #
+                  {tag}
+                </span>
```

Comment:
사소하지만 이렇게 작성하면 문자 + 변수를 한 줄에 깔끔하게 쓸 수 있어요
```suggestion
                <span className={styles['search-rolling-banner__tag-style']}>
                  {`#${tag}`}
                </span>
```